components/monitoring/prometheus/README.md

```diff
@@ -107,14 +107,77 @@ namespaces).
 For those reasons, another instance is needed to federate the other instances, and
 write metrics to RHOBS.
 
-This instance collect selected metrics from Platform Prometheus and UWM Prometheus, and
+This instance collects selected metrics from Platform Prometheus and UWM Prometheus, and
 remote-writes selected labels for those metrics to RHOBS, which in turn, makes the
 metrics accessible to AppSRE Grafana.
 
 This Prometheus instance is deployed using a MonitoringStack custom resource provided
 by the Observability Operator. This operator is installed by default in Production and Staging clusters.  
 In Development clusters, it's not installed by default to prevent conflicts with other deployments. 
 It can be installed and configured in development by using the `--obo/-o` flags.  
-For example:  
+For example:
+
 `./hack/bootstrap-cluster.sh preview --obo`  
 `./hack/bootstrap-cluster.sh preview -o`
+
+#### Federation and Remote-write
+
+Through Federation and remote-write configurations, only a subset of the metrics and
+the labels collected within the data plane clusters reach RHOBS. For that reason, it
+might be that metrics that are visible via the OCP web console (under Observe -->
+Metrics) do not reach RHOBS and are not visible in AppSRE Grafana.
+
+The Platform Prometheus instance monitors a wide variety of resources which are, in
+nature, of an unbound cardinality (e.g. containers, pods, jobs). Consequently, it
+generates a substantial amount of metrics time series that cannot all be forwarded to
+RHOBS. For that reason, we only allow a very small subset of the metrics it scrapes
+to be federated by the OBO Prometheus instance (and later remote-written to RHOBS).
+
+The UWM Prometheus instance, on the other hand, generates a very few time series by
+default, and the metrics it is configured to scrape for services are generally of
+constant cardinality. I.e. the number of time series stored for a given service does
+not grow based on the service load. For this reason, we allow all metrics it scrapes
+to be federated by the OBO Prometheus instance (and reach RHOBS).
+
+All **metrics** reaching the OBO instance are remote-written to RHOBS, but not all
+**labels** are. This means that it might be that time series visible in AppSRE Grafana
+will not include some of the labels the same time series have on the data plane
+clusters. The OBO instance is configured to remote-write only specific labels, and if
+the presence of a new label is required in AppSRE, then this label should be added to
+the configurations.
+
+##### Troubleshooting Missing Metrics and Labels
+
+There are a few steps to follow when specific metrics or labels are required for new
+alerting rules or Grafana dashboards, but are not present in AppSRE Grafana.
```

Comment:
```suggestion
alerting rules or Grafana dashboards, but are not present in RHOBS.
```

Reply:
The thing here is that there's no easy way for us to see if the metrics/labels are reaching RHOBS, so we check AppSRE Grafana instead, assuming that the issue is not between RHOBS and AppSRE Grafana.

Reply:
Ah, good point. LGTM then :+1:

Reply:
I'll add a note

Reply:
@amisstea please take another look

Reply:
Even better! Still LGTM